python/ray/serve/_private/deployment_scheduler.py

```diff
@@ -519,13 +527,27 @@ def _schedule_replica(
                 if scheduling_request.placement_group_strategy
                 else "PACK"
             )
-            pg = self._create_placement_group_fn(
-                scheduling_request.placement_group_bundles,
-                placement_group_strategy,
-                _soft_target_node_id=target_node_id,
-                lifetime="detached",
-                name=scheduling_request.actor_options["name"],
-            )
+            try:
+                pg = self._create_placement_group_fn(
+                    scheduling_request.placement_group_bundles,
+                    placement_group_strategy,
+                    _soft_target_node_id=target_node_id,
+                    lifetime="detached",
+                    name=scheduling_request.actor_options["name"],
+                )
+            except Exception:
```

Comment:
Non-blocker: It would be nice to catch the exact exception If we know exactly the error this is going to throw.

Reply:
Agreed, but the purpose of this check is to be extra defensive. If Ray core silently changes its implementation to raise a new type of exception, this clause should still catch it. Otherwise, the controller enters a bad state where it busy loops due to the failed PG scheduling.

Reply:
I synced with @jjyao, and he also felt catching a blanket exception is fine for now. I agree that we should typically avoid it though for all the reasons you listed. I'll leave it as-is.